Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: generate
+generate:
+	docker build -t proto-gen .
+	docker run -v $(CURDIR)/proto:/app/proto -v $(CURDIR)/go:/app/go -v $(CURDIR)/js:/app/js -v $(CURDIR)/python:/app/python -v $(CURDIR)/java:/app/java proto-gen
```

Comment:
I am not sure if `$(CURDIR)` is cross platform? I would suggest to use `$(shell pwd)`

Reply:
`$(shell pwd)` is a better alternative. Done